src/app/components/Application/Application.jsx

```diff
@@ -4,61 +4,88 @@ import PropTypes from 'prop-types';
 // NYPL Components
 import { Header, navConfig } from '@nypl/dgx-header-component';
 import Footer from '@nypl/dgx-react-footer';
+
 import Hero from '../Hero/Hero.jsx';
 import AgeTabs from '../AgeTabs/AgeTabs.jsx';
 import Books from '../Books/Books.jsx';
 import Sidebar from '../Sidebar/Sidebar.jsx';
+import BookStore from '../../stores/BookStore.js';
 
-const App = (props) => {
-  const annualList = !!(props.params && props.params.type &&
-    (props.params.type === 'childrens' || props.params.type === 'ya'));
-  let mobileAboutLink = null;
-
-  if (annualList) {
-    mobileAboutLink = (
-      <div className="mobile-about">
-        <span className="mobile-about-divider"></span>
-        <h2 className="mobile-about-link">
-          <a href="http://nypl.org/books-music-dvds/recommendations/about/annual-lists">
-            About this list
-          </a>
-        </h2>
-      </div>
-    );
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = BookStore.getState();
+    this.onChange = this.onChange.bind(this);
   }
 
-  return (
-    <div className="home">
-      <Header
-        navData={navConfig.current}
-        skipNav={{ target: 'app-content' }}
-      />
-      <Hero {...props} annualList={annualList} />
-      <div id="app-content">
-        {props.children}
-
-        {!annualList && (<AgeTabs />)}
-
-        <div className="main-container">
-          <div id="sidebar">
-            <Sidebar filters={props.filters} {...props} annualList={annualList} />
-          </div>
-          <div id="books">
-            <Books {...props} annualList={annualList} />
-          </div>
+  componentDidMount() {
+    BookStore.listen(this.onChange);
+  }
+
+  componentWillUnmount() {
+    BookStore.unlisten(this.onChange);
+  }
+
+  onChange() {}
+
+  render() {
+    const annualList = !!(this.props.params && this.props.params.type &&
+      (this.props.params.type === 'childrens' || this.props.params.type === 'ya'));
+    let mobileAboutLink = null;
 
-          {mobileAboutLink}
+    if (annualList) {
```

Comment:
Would be cleaner to have a function that renders the about mobile link. Would be re-usable if needed and can verify the existence of annual list prop to return. The href should also be part of the config file, in the event it is changed, we have one place to make modifications.